addons/portal/static/src/chatter/frontend/composer_patch.js

```diff
@@ -0,0 +1,12 @@
+import { Composer } from "@mail/core/common/composer";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(Composer.prototype, {
+    setup() {
+        super.setup();
+        if (!this.thread || this.thread.model !== "discuss.channel") {
```

Comment:
We need to change all of those `this.thread.model !== "discuss.channel"` something.

I don't know what is the goal, but it's highlighting bad design of something.